_posts/2022-11-01-ndk_advanced_custom_fields-CVE-2022-40840.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "[CVE-2022-40840][CVE-2022-40841] ndk design NdkAdvancedCustomizationFields 3.5.0 is vulnerable to Cross Site Scripting (XSS) via createPdf.php"
+categories: module
+author:
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,ndkadvancedcustomizationfields"
+severity: "medium (6.8)"
+---
+
+Ndk design NdkAdvancedCustomizationFields 3.5.0 is vulnerable to Reflected Cross Site Scripting (XSS-T1 of category 1) via createPdf.php and showPreview.php.
+
+## Summary
+
+* **CVE ID**: [CVE-2022-40840](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-40840) [CVE-2022-40841](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-40841)
+* **Published at**: 2022-11-01
+* **Advisory source**: [github](https://github.com/daaaalllii/cve-s/blob/main/CVE-2022-40840/poc.txt)
+* **Vendor**: PrestaShop
+* **Product**: NdkAdvancedCustomizationFields
+* **Impacted release**: <= 3.5.0
+* **Product author**: 
+* **Weakness**: [CWE-79](https://cwe.mitre.org/data/definitions/79.html)
+* **Severity**: medium (6.8)
+
+## Description
+
+Since we do not have access to the module, we cannot confirm the XSS type, but based on the POC and until proven otherwise, Ndk design NdkAdvancedCustomizationFields 3.5.0 is vulnerable to Reflected Cross Site Scripting (XSS-T1 of category 1) via createPdf.php.
+
+Be warned that, always based on POC given, the second exploit against showPreview.php could be a [XSS T2 - potentially F2B)(https://security.friendsofpresta.org/modules/2023/02/07/stored-xss.html) - if it's confirmed, this exploit could be a critical vulnerability CVSS 3.1 : 9.6/10 (AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:H/A:N).
```

Comment:
Be warned that, based on the provided POC, [...]
[...] If confirmed, this exploit could pose a critical vulnerability [...]